includes/rest-api.php

```diff
@@ -603,6 +683,80 @@ function pmpro_rest_api_get_checkout_levels( $request ) {
 			return new WP_REST_Response( $r );
 		}
 
+
+		/// ZAPIER TRIGGERS
+		/**
+		 * Function used for testing authentication through Zapier API.
+		 * @since 2.7
+		 */
+		public function validate_me( $request ) {
+
+			$params = $request->get_params();
+
+			if ( is_user_logged_in() ) {
+			  $me = wp_get_current_user()->display_name;
+			} else {
+				$me = false;
+			}
+		
+			wp_send_json_success( array( 'username' => $me ) );
+		}
+
+
+		function pmpro_rest_api_recent_memberships( $request ) {
```

Comment:
```suggestion
		/**
		 * Handle requests for the list of recent memberships.
		 *
		 * @since 2.6.0
		 */
		public function pmpro_rest_api_recent_memberships( $request ) {
```